src/sentry/models/sentryappavatar.py

```diff
@@ -1,11 +1,26 @@
 from enum import Enum
+from typing import TYPE_CHECKING, Iterable, Mapping, Union
 
 from django.db import models
 
 from sentry.db.models import FlexibleForeignKey
 
 from . import AvatarBase
 
+if TYPE_CHECKING:
+    from sentry.models import SentryApp
+
+
+def get_sentry_app_avatars(sentry_app: "SentryApp") -> Iterable[Mapping[str, Union[str, bool]]]:
```

Comment:
I'd actually advocate against putting this function on the model. In MVC this JSON is part of the _View_ and should stay in the serializer layer.

Reply:
@mgaeta The function isn't an issue though right, I should just move it to a serializer?

Reply:
In fact I'd take advantage of Django here. In `sentryappavatar.py` you have:
```py
sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="avatar")
```
So in the function can do: 
```py
return [
    {
        "avatarType": img.get_avatar_type_display(),
        "avatarUuid": img.ident,
        "color": img.color,
    }
    for img in sentry_app.avatar
]
```
I'd recommend changing the `related_name` to `avatars`.

Reply:
Ok sorry this is coming in as parts but you should also consider creating a new file with just 
```py
@register(SentryAppAvatar)
class SentryAppAvatarSerializer(Serializer):
    def serialize(self, obj, attrs, user, access):
        return {
            "avatarType": obj.get_avatar_type_display(),
            "avatarUuid": obj.ident,
            "color": obj.color,
        }
```
Then the code becomes:
```py
return [serialize(avatar, attrs, user, access) for avatar in sentry_app.avatar]
```

Reply:
I believe I implemented your suggestions in 92a0cc598d35784c6704245b8541f6afd112350f